app.py

```diff
@@ -8,6 +8,7 @@
 defaultWorkPath = "work"
 defaultBuildRoot = "/tmp/preview-bot/builds"
 defaultRemoteUrl = "https://github.com/CSCfi/csc-user-guide"
+defaultSiteURL = "https://docs.csc.fi/"
```

Comment:
Why is this the production URL?

Reply:
No particular reason. I can set it to any value.

This is only to replace it later in line 114. I imagine it would cleaner to use the actual preview URL (by setting it in env)?

Reply:
When I don't know exactly how something works, I start to worry and, for example, this sort of hypothetical scenario starts to form in my mind: MkDocs somehow uses that URL when building a preview instance and then a user clicks on a link taking them to the production site while they think they are still on preview.

Reply:
Fair enough, I will change it now.